python/lsst/daf/butler/core/utils.py

```diff
@@ -53,6 +55,10 @@
 from lsst.utils import doImport
 
 
+# regular expression that can be used to find supported fits file extensions.
+fits_re = r"\.fit[s]?\b"
```

Comment:
I don't think the default regex for fits files should be in daf_butler. daf_butler knows nothing about fits files. It's only specifically obs_base raw ingest that wants to find fits files. The default should be `.*`